hack/run-e2e.yml

```diff
@@ -120,6 +120,26 @@
             source .test.env && \
             make e2e-local
           register: run_tests
+                
+        - name: Find report.xml file on remote server
+          find:
+            paths: "~/go-metadata-{{ run_id }}"
+            patterns: "*_go_metadata_test_report.xml"
+            file_type: file
+          register: report_files
+
+        - name: Extract filename from the found files
+          set_fact:
+            filename: "{{ report_files.files | first | default({}) }}"
+          when: report_files.files | length > 0
+
+        - name: Copy report.xml to local folder directory
+          fetch:
+            src: "{{ filename.path }}"
```

Comment:
Can we skip `Extract filename from the found files` and reference `src` to the the filename directly? I think the fact `filename` won't be reused?